bundown/src/plugin/example.md

```diff
@@ -0,0 +1,48 @@
+# example
+
+```shell
+echo "Hello from example.md"
+```
+
+```typescript -t time
+console.log('This only runs with --tag "time"', Date.now())
+```
+
+```typescript -f ./src/plugin/test.ts
+console.log(`This won't run, since it uses -f`)
+```
+
+```typescript --os mac --os linux
+console.log(`This only runs on mac or linux`)
+```
+
+```python
+print("Python works too!")
+```
+
+```go
+package main
+
+import "fmt"
+
+func main() {
+	fmt.Println("Go works too!")
```

Comment:
Please change this tab to 2 or 4 spaces for now - the actual fix here should be a follow-up PR but currently it seems that tabs break our box drawing in print mode, so we should find and replace them with a set number of spaces when code is printed *before* string widths are calculated etc.

Reply:
(When I say follow up PR in this case I just mean a future fix, doesn't necessarily have to be anybody specific implementing it or on any timeline, I don't consider it an urgent issue.)

Reply:
Ah, I think the tabs were because of this just being copy-pasted from default go "hello world". I will fix.